pylama/lint/pylama_pydocstyle.py

```diff
@@ -22,4 +27,4 @@ def run(path, code=None, **meta):
                      if e.message[4] == ':' else e.message),
             'type': 'D',
             'number': e.code
-        } for e in PEP257Checker().check_source(code, path)]
```

Comment:
I had to add the third argument to `check_source` (new pydocstyle takes at least 3). Otherwise, pylama swallows swallow the exception and disables the linter - that's why the CI fails on undetected errors.

(as in the diff: https://github.com/klen/pylama/issues/96#issuecomment-295673453)